documentation/_puppetdb_nav.html

```diff
@@ -4,6 +4,7 @@ <h2 id="puppetdb">PuppetDB {{ page.doc.version }}</h2>
   <li><strong>General information</strong>
     <ul>
       <li><a href="{{puppetdb}}/index.html">Overview and requirements</a></li>
+      <li><a href="contributing.html">Contributing to PuppetDB</a></li>
```

Comment:
You'll want the `{{puppetdb}}` it's the path to our docs although i'm not sure where it's defined.